eZ/Publish/Core/Persistence/Legacy/Content/UrlAlias/Handler.php

```diff
@@ -441,6 +441,15 @@ protected function createUrlAlias($action, $path, $forward, $languageCode, $alwa
                 $topElementMD5,
                 $data
             );
+        // adding existing alias to diffrent laquage version
+        } elseif ($row['action'] == $action && !((int)$row['lang_mask'] & $languageId)) {
```

Comment:
technically result of a bit-wise AND is not Boolean, so instead negating it, you should:
```suggestion
        } elseif ($row['action'] == $action && 0 === ((int)$row['lang_mask'] & $languageId)) {
```